backend/.env.development

```diff
@@ -0,0 +1,2 @@
+FRONT_URL="http://localhost:3000"
+JWT_SECRET="s3cr3t"
```

Comment:
This file is intended only for development (local) environments, so no problem sharing these variables. Having it on the repo will help other devs setup faster.